atlassian/bitbucket.py

```diff
@@ -139,16 +133,16 @@ def project_keys(self, key, limit=99999, filter_str=None):
         :param filter_str:  OPTIONAL: users filter string
         :return:
         """
-        if not self.cloud:
-            url = 'rest/keys/1.0/projects/{key}/ssh'.format(key=key)
-        else:
-            url = 'rest/keys/2.0/projects/{key}/ssh'.format(key=key)
+        resource = self.resource_url('projects/{key}/ssh'.format(key=key))
```

Comment:
Here ressource_url MUST not be used, url is .../rest/keys/... and not .../rest/api/...